Dockerfile.rhel7

```diff
@@ -0,0 +1,30 @@
+FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.12 AS builder
+WORKDIR /go/src/github.com/openshift/machine-config-operator
+COPY . .
+RUN WHAT=machine-config-operator ./hack/build-go.sh; \
+    mkdir -p /tmp/build; \
+    cp /go/src/github.com/openshift/machine-config-operator/_output/linux/$(go env GOARCH)/machine-config-operator /tmp/build/machine-config-operator
+RUN WHAT=machine-config-daemon ./hack/build-go.sh; \
+    mkdir -p /tmp/build; \
+    cp /go/src/github.com/openshift/machine-config-operator/_output/linux/$(go env GOARCH)/machine-config-daemon /tmp/build/machine-config-daemon
+RUN WHAT=machine-config-controller ./hack/build-go.sh; \
+    mkdir -p /tmp/build; \
+    cp /go/src/github.com/openshift/machine-config-operator/_output/linux/$(go env GOARCH)/machine-config-controller /tmp/build/machine-config-controller
+RUN WHAT=machine-config-server ./hack/build-go.sh; \
+    mkdir -p /tmp/build; \
+    cp /go/src/github.com/openshift/machine-config-operator/_output/linux/$(go env GOARCH)/machine-config-server /tmp/build/machine-config-server
+RUN WHAT=setup-etcd-environment ./hack/build-go.sh; \
+    mkdir -p /tmp/build; \
+    cp /go/src/github.com/openshift/machine-config-operator/_output/linux/$(go env GOARCH)/setup-etcd-environment /tmp/build/setup-etcd-environment
+
+FROM registry.svc.ci.openshift.org/ocp/4.0:base
```

Comment:
Another unrelated question...anyone know what's the status of switching to the UBI image, specifically UBI8?  If we did that I think we wouldn't need to duplicate the two Dockerfiles at all even.

But still not a blocker for this just noting the question.

Reply:
not sure. fwiw: i did double check that this change worked for ART, but that was all.

Reply:
This is a good one, probably worth following up with openshift/release as I imagine it would be projects wide

Reply:
I can followup on this if you'd  like @runcom

Reply:
sure thing, as Colin said, not blocking